cli/pkg/local/app.go

```diff
@@ -16,10 +17,14 @@ import (
 	"github.com/rilldata/rill/cli/pkg/web"
 	"github.com/rilldata/rill/runtime"
 	"github.com/rilldata/rill/runtime/compilers/rillv1beta"
+
+	// Register some standard stuff
 	_ "github.com/rilldata/rill/runtime/connectors/gcs"
 	_ "github.com/rilldata/rill/runtime/connectors/https"
 	_ "github.com/rilldata/rill/runtime/connectors/s3"
 	"github.com/rilldata/rill/runtime/drivers"
+
+	// Register some standard stuff
```

Comment:
See earlier comment, let's discuss what to do here

Reply:
this is `blank-imports` of `revive` linter, should ignore this?